core/src/test/scala/unit/kafka/server/AbstractFetcherThreadTest.scala

```diff
@@ -814,7 +814,8 @@ class AbstractFetcherThreadTest {
     extends AbstractFetcherThread("mock-fetcher",
       clientId = "mock-fetcher",
       sourceBroker = new BrokerEndPoint(leaderId, host = "localhost", port = Random.nextInt()),
-      failedPartitions) {
+      failedPartitions,
+      brokerTopicStats = null) {
```

Comment:
nit: any harm constructing a `BrokerTopicStats` object?

Reply:
Not at all, will do it.